cli/cli/kurtosis_config/overrides_objects/v5/grafloki_config_v5.go

```diff
@@ -0,0 +1,7 @@
+package v5
+
+type GraflokiConfig struct {
```

Comment:
Same comment - IMO this would be better as `GrafanaLokiConfig`, or maybe even better, `InbuiltLoggingConfig` or something explaining the purpose

Reply:
renaming to GrafanaLokiConfig